tiptapy/__init__.py

```diff
@@ -161,11 +161,9 @@ def inner_render(self, node):
         language = attrs.get("language", "")
         content = node.get("content", {})[0]
         text = e(content.get("text", ""))
-        html = ""
+        html = f'<div><pre><code>{text}</code></pre></div>'
```

Comment:
Do we want to add "# noqa: E501" here?
Not sure how it works.

Reply:
@rajatsingla the default PEP8 line length is 79 Chars[0]. `noqa: E501` is just a way of saying that ignore this line while checking PEP8 rules. Since this line is not going out of chars so we don't need to add `noqa: E501`

`E501` is just one of a PEP8 errors for line too long.


[0] https://www.python.org/dev/peps/pep-0008/#maximum-line-length